packages/datetime/src/components/DatePickerCarousel/carouselCalendar/__test__/dayCard.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import mountWithApplication from '../../../../helpers/mountWithApplication';
+import DayCard from '../dayCard';
+import { StyledDisabledDayCard } from '../styled';
+
+describe('<DayCard />', () => {
+    it('should render the right day', () => {
+        const date = new Date('2020/10/21');
+        const component = mountWithApplication(<DayCard date={date} />);
+        expect(component.find('span').at(0).text()).toBe('Wed');
+        expect(component.find('span').at(1).text()).toBe('21');
+    });
+    it('should call onChange with the right date when click the button', () => {
+        const date = new Date('04/24/2019');
+        const onChangeMockFn = jest.fn();
+        const component = mountWithApplication(<DayCard date={date} onChange={onChangeMockFn} />);
+        component.find('button').simulate('click');
+        expect(onChangeMockFn).toHaveBeenCalledWith(new Date('04/24/2019'));
```

Comment:
this test is repeated in the above three test files, I think we should keep only the first one in the main component since it is the final result we want